### PR TITLE
feat: incoming payment resolver

### LINKED ISSUE #1343: Add GraphQL admin API query to get incoming payment by id
- Implement:
```gql
type Query {
    incomingPayment(id: String!): IncomingPayment
}
```
similar to the outgoing payment query

- Add API to Postman collection

### PR DESCRIPTION
**Pull request by BlairCurrey**
## Changes proposed in this pull request

- adds `incomingPayment` resolver and unit test
- also added Get Incoming Payment request to Postman's `Rafiki Admin APIs`: https://www.postman.com/interledger/workspace/interledger/request/22855701-28f21533-1808-483e-ba36-4bd11ddba644

## Context

fixes #1343 

## Checklist

- [X] Related issues linked using `fixes #number`
- [X] Tests added/updated
- [x] Documentation added
- [x] Make sure that all checks pass

### CODE REVIEW

**Comment by BlairCurrey** [comment]
Do these docs automatically get updated or is there anything I need to do here? https://rafiki.dev/docs/reference/queries It seems

**Comment by mkurapov** [comment]
> Do these docs automatically get updated or is there anything I need to do here? https://rafiki.dev/docs/reference/queries

They should trigger a workflow on https://github.com/interledger/rafiki.dev to autogenerate docs and open a PR in that repo with the changes

File: packages/backend/src/graphql/schema.graphql [hunk 1]
```diff
@@ -37,6 +37,9 @@ type Query {
 
   "Fetch an Open Payments outgoing payment"
   outgoingPayment(id: String!): OutgoingPayment
+
+  "Fetch an Open Payments incoming payment"
+  incomingPayment(id: String!): IncomingPayment
```
Comment (mkurapov) [inline packages/backend/src/graphql/schema.graphql:42]: I wrote out the type definition for this in the original issue (and this isn't specific to this PR), but looking at the schema in general, I think going forward we should use `id: ID!` type for our id's. It doesn't _actually_ change anything, except for readability, but just makes the schema tidier IMO

**Review by mkurapov — APPROVED** [review]
Looks good 🚀

File: packages/backend/src/graphql/schema.graphql (hunk 1 above)
Comment (BlairCurrey) [reply packages/backend/src/graphql/schema.graphql:42]: That makes sense

**Pull request merged**